Stellar-ledger.x

```diff
@@ -506,12 +509,26 @@ struct LedgerCloseMetaExtV1
     int64 sorobanFeeWrite1KB;
 };
 
+struct LedgerCloseMetaExtV2
```

Comment:
Shall we make this extensible? Extension nesting is not super readable, but I think it's a better option for the flag-protected meta changes in minor releases.